docs/rules/camelcase.md

```diff
@@ -12,10 +12,12 @@ This rule has an object option:
 
 * `"properties": "always"` (default) enforces camelcase style for property names
 * `"properties": "never"` does not check property names
+* `"ignoreDestructuring": false` (default) enforces camelcase style for destructured identifiers
+* `"ignoreDestructuring": true` does not check destructured identifiers
 
-### always
+### Defaults (properties: "always", "ignoreDestructuring": false)
```

Comment:
Should the docs about different options be separated into different sections?

Reply:
I agree with this. Let's have sections for `properties` and `ignoreDestructuring` options separately, and create subsections for the always/never and true/false cases (respectively) for each.